### ISSUE TITLE
Animated GIF: cannot be paused/run only once

### ISSUE THREAD

**Issue by 2019-05-10** [mattermost/mattermost-server#20468]
**Summary**
GIFs once inserted run infinitely, thus distrcting/annoying users.

**Environment**
- Operating System:
- Mattermost Desktop App version: 5.1.0
- Mattermost Server version: X.X.X 

**Steps to reproduce**
have a coworker insert an animted GIF into an ongoing chat
GIF wriggles and blinks continuously

**Expected behavior**
run only once or allow to pause/stop

**Observed behavior**
runs and runs, cannot be paused nor stopped

**Comment by amyblais** [comment]
Opened https://mattermost.atlassian.net/browse/MM-45095.

**Comment by hmhealey** [comment]
Thanks for the report! We have another ticket to add this [here](https://github.com/mattermost/mattermost-server/issues/15399).
